cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HTableNameConverter.java

```diff
@@ -16,18 +16,113 @@
 
 package co.cask.cdap.data2.util.hbase;
 
+import co.cask.cdap.common.conf.CConfiguration;
+import co.cask.cdap.common.conf.Constants;
+import co.cask.cdap.data2.util.TableId;
+import co.cask.cdap.proto.Id;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Joiner;
+import com.google.common.base.Preconditions;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
 /**
  * Common utility methods for dealing with HBase table name conversions.
  */
 public abstract class HTableNameConverter {
+  protected static final String HBASE_NAMESPACE_PREFIX = "cdap_";
```

Comment:
So this PR doesn't use the CConf for hbase namespace prefix, but only in the tablename (for backwards compatability).
It should be only few changes to do that though.